script.js

```diff
@@ -1,4 +1,111 @@
+// overrides: reverse?
+var gameMode = "waiting for username"; // can also be set to 'reverse' by user override
+var username = "";
+var playerScore = 0;
+var computerScore = 0;
+var drawScore = 0;
+
+var moveSet = {
+  1: "scissors",
+  2: "paper",
+  3: "stone", // every move beats the move below it.
+};
```

Comment:
Nice job on using an object to store the moveset, and that each key beats the key below it which is a pretty ingenious idea